src/tests/ftest/recovery/start_subcommand.py

```diff
@@ -0,0 +1,164 @@
+"""
+  (C) Copyright 2024 Intel Corporation.
+  (C) Copyright 2025 Hewlett Packard Enterprise Development LP
+
+  SPDX-License-Identifier: BSD-2-Clause-Patent
+"""
+import time
+
+from general_utils import check_file_exists
+from recovery_test_base import RecoveryTestBase
+from run_utils import run_remote
+
+
+class StartSubcommandTest(RecoveryTestBase):
+    """Test dmg check start subcommands.
+
+    Subcommand list. (Not all subcommands are tested here.)
+    -n, --dry-run           Scan only; do not initiate repairs.
+    -r, --reset             Reset the system check state.
+    -f, --failout=[on|off]  Stop on failure.
+    -a, --auto=[on|off]     Attempt to automatically repair problems.
+    -O, --find-orphans      Find orphaned pools.
+    -p, --policies=         Set repair policies.
+
+    :avocado: recursive
+    """
+
+    def test_reset(self):
```

Comment:
Can we name this something more globally unique? `test_reset` is very ambiguous in the context of DAOS. Maybe something to match the command being tested?
```suggestion
    def test_dmg_check_start(self):
```